setup.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python3
+
+from setuptools import setup, find_packages
+
+setup(
+    name="canonicalwebteam.discourse",
```

Comment:
this is a mistake from me, I was going to call it `canonicalwebteam.docs` to have the same pattern as `canonicalwebteam.blog` where we don't specify the source

Reply:
E.g.: https://github.com/canonical-websites/docs.snapcraft.io/issues/105 - this will need an entirely separate solution, that I think should be written separately and kept elsewhere.

Reply:
ok I am good with that naming,
done